code/datums/controllers/event_controls.dm

```diff
@@ -55,6 +55,11 @@ var/datum/event_controller/random_events
 			special_events += RE
 
 	proc/process()
+		if (emergency_shuttle.location > SHUTTLE_LOC_STATION)
```

Comment:
this should also check for post-round game status i think

Reply:
Where's that status stored?

Reply:
`current_state = GAME_STATE_FINISHED`